### DISCUSSION TITLE
Welcome to forum Discussions!

### DISCUSSION THREAD

**Discussion by chinchang** [cssbattle/forum#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by duarch** [comment]
Hi guys! Are you planning to use wiki also to put all the tips in an organized way?

**Reply by chinchang** [reply]
Not sure if you already know, but we do have a Tips page -> https://cssbattle.dev/tips
Or are you talking about having those same tip in the Wiki here too? If yes, any suggestion on organizing them better?
